### PR TITLE
kata 3 solution

### PR DESCRIPTION
**Pull request by RTXVader**

### CODE REVIEW

File: controller/app.controller.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
-     return   res.status(200).send(topics)
-    })
-}
\ No newline at end of file
+  return gettingTopics().then((topics) => {
+    return res.status(200).send({ topics: topics });
+  });
+};
+exports.getDescriptions = (req, res) => {
+  console.log("controller");
```
Comment (hevdevs) [inline controller/app.controller.js:9]: remove console.log

File: __tests__/app.test.js [hunk 2]
```diff
@@ -1,42 +1,54 @@
...
   });
 });
+
+describe("GET /api", () => {
+  it("200: should return an object describing all the available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        console.log(body);
```
Comment (hevdevs) [inline __tests__/app.test.js:50]: remove console.log

**Review by hevdevs — COMMENTED** [review]
Just remove console.logs as highlighted otherwise looking good

**Comment by RTXVader** [comment]
Ok ty

**Pull request merged**